scripts/init_dev

```diff
@@ -38,10 +38,15 @@ sudo apt-get update -q
 # sudo apt-get -y upgrade
 
 # Install pip and couchdb (couchdb is not available via rosdep -- see below)
-echo "Install pip, couchdb and boostrapping tools"
-sudo apt-get install -y -q python-pip couchdb
+echo "Install pip, couchdb and bootstrapping tools"
+sudo apt-get install -y -q couchdb
+sudo easy_install pip
```

Comment:
@novemberalpha can you clarify? Was `python-pip` flakey or was `sudo easy_install pip` flakey? How did it fail?